kits/survey-demo/migrations/2_deploy.js

```diff
@@ -0,0 +1,223 @@
+const PublicResolver = artifacts.require('@aragon/os/contracts/lib/ens/PublicResolver')
+const Repo = artifacts.require('@aragon/os/contracts/apm/Repo')
+
+const SurveyKit = artifacts.require('@aragon/kits-survey/contracts/SurveyKit')
+const Survey = artifacts.require('@aragon/apps-survey/contracts/Survey')
+
+// Utils
+const surveyAppId = namehash('survey.aragonpm.eth')
+const surveyKitEnsNode = namehash('survey-template.aragonpm.eth')
+const timeTravel = require('@aragon/test-helpers/timeTravel')(web3)
+const pct16 = x => new web3.BigNumber(x).times(new web3.BigNumber(10).toPower(16))
+const createdSurveyDao = receipt => receipt.logs.filter(x => x.event == 'DeployInstance')[0].args.dao
+const createdSurveyId = receipt => receipt.logs.filter(x => x.event == 'StartSurvey')[0].args.surveyId
+const installedApp = (receipt, appId) => receipt.logs.filter(x => x.event == 'InstalledApp' && x.args.appId === appId)[0].args.appProxy
+
+// Survey params
+const SURVEY_DURATION = 60 * 60 * 16 // 24 hours
+const SURVEY_PARTICIATION = pct16(10) // 10%
+const SURVEY_CHART_BLOCKS = 16 // front-end chart blocks
+const TOKEN_BASE_DECIMAL = 1e18
+
+module.exports = async (deployer, network, accounts) => {
+  const root = accounts[0]
+
+  const ens = ENS.at(
+    process.env.ENS ||
+    '0x5f6f7e8cc7346a11ca2def8f827b7a0b612c56a1' // aragen's default ENS
+  )
+  const surveyKitRepo = Repo.at(
+    await PublicResolver.at(
+      await ens.resolver(surveyKitEnsNode)
+    ).addr(surveyKitEnsNode)
+  )
+  // Contract address is second return of Repo.getLatest()
+  const surveyKit = SurveyKit.at((await surveyKitRepo.getLatest())[1])
+
+  // Create minime token and assign to accounts
+  const minimeFac = await MiniMeTokenFactory.new()
+  const surveyToken = await MiniMeToken.new(
+    minimeFac.address,
+    '0x00',
+    '0x00',
+    'Demo Survey',
+    18,
+    'SUR',
+    true
+  )
+
+  // Total of 30 tokens being assigned
+  const tokenAssignments = [
+    TOKEN_BASE_DECIMAL * 16,
+    TOKEN_BASE_DECIMAL * 8,
+    TOKEN_BASE_DECIMAL * 4,
+    TOKEN_BASE_DECIMAL * 2
+  ]
+  await Promise.all(
+    tokenAssignments.map((amount, i) => {
+      const account = accounts[i]
+      console.log(`Assigned ${account} ${amount} tokens`)
+      return surveyToken.generateTokens(accounts[i], amount)
+    })
+  )
+
+  // Create DAO with just Survey installed
+  console.log('Creating Survey DAO...')
+  const surveyDaoReceipt = await surveyKit.newInstance(surveyToken.address, root, '0x00', SURVEY_DURATION, SURVEY_PARTICIATION)
+  const surveyDaoAddr = createdSurveyDao(surveyDaoReceipt)
+  const surveyAppAddr = installedApp(surveyDaoReceipt, surveyAppId)
+
+  // Create some sample surveys and vote on them
+  const survey = Survey.at(surveyAppAddr)
+
+  const newSurvey = ({ question, description, options }) => {
+    const metadata = {
+      specId: namehash('1.metadata.survey.aragonpm.eth'),
+      metadata: {
+         question,
+         description,
+         options,
+      }
```

Comment:
@bpierre @izqui You'll see I've made a few modifications on the [original spec](https://github.com/aragon/aragon-apps/issues/231#issue-310786634):

- `specId`: namehash-based value
- `metadata`: removed `descriptionSource` and `tags` for now